lib/index.ts

```diff
@@ -792,4 +807,12 @@ export default class Backburner {
 
     this._autorun = true;
   }
+
+  private createTask(queueName, method){
+    if(this.ASYNC_STACKS && console["createTask"]){
```

Comment:
`prettier` on demand 😝 

```suggestion
    if (this.ASYNC_STACKS && console["createTask"]) {
```